Dockerfile

```diff
@@ -29,6 +30,7 @@ RUN apt-get update && apt-get dist-upgrade -y && apt-get -y install \
 
 # Build time variables
 ENV LSMB_VERSION 1.5.8
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
In theory, it can be set once before the first RUN statement, but for now I've simply made this file the same as master in that regard.
Later we can change how it's done in master and 1.5.
I'd rather keep them as close to the same as possible to reduce merge conflicts